pkg/controller/command/presentationclient/command.go

```diff
@@ -52,6 +56,11 @@ func (c *Command) GetHandlers() []command.Handler {
 }
 
 func (c *Command) SavePresentation(_ io.Writer, req io.Reader) command.Error {
+	if c.sdsComm.SDSServerURL == "" {
+		logutil.LogError(logger, commandName, savePresentationCommandMethod, failToStorePresentationNoSDSURLSet)
+		return command.NewValidationError(InvalidRequestErrorCode, errFailToStorePresentationNoSDSURLSet)
+	}
```

Comment:
This is just an extra layer of safety in case this method gets called somehow (perhaps by another dev in the future). Without this check, the error message is super confusing.